test/stats.bats

```diff
@@ -67,20 +62,20 @@ function teardown() {
     run crictl stats -o json "$ctr1_id"
     echo "$output"
     [ "$status" -eq 0 ]
-    ctr1_stats_JSON="$output"
+    jq -e '.stats[0].attributes.id == "'$ctr1_id'"' <<< "$output"
+    run jq -e '.stats[0].memory.workingSetBytes.value' <<< "$output"
+    [ "$status" -eq 0 ]
+    ctr1_memory_bytes="$output"
 
-    run crictl stats -o json "$crt2_id"
+    run crictl stats -o json "$ctr2_id"
     echo "$output"
     [ "$status" -eq 0 ]
     ctr2_stats_JSON="$output"
-
-    run echo $ctr1_stats_JSON | jq -e '.stats[0].memory.workingSetBytes.value'
-    [ "$status" -eq 0 ]
-    ctr1_memory_bytes="$output"
-    run echo $ctr2_stats_JSON | jq -e '.stats[0].memory.workingSetBytes.value'
+    jq -e '.stats[0].attributes.id == "'$ctr2_id'"' <<< "$output"
```

Comment:
we lost ctr1_memory_bytes here
I think these would be better if we just did 
```
    ctr1_stats_JSON=$(crictl stats -o json --id "$ctr1_id")
    ctr2_stats_JSON=$(crictl stats -o json --id "$ctr2_id")
    ctr1_memory_bytes=$(echo $ctr1_stats_JSON | jq -e '.stats[0].memory.workingSetBytes.value')
    ctr2_memory_bytes=$(echo $ctr2_stats_JSON | jq -e '.stats[0].memory.workingSetBytes.value')
```
maybe echoing them between?

Reply:
> we lost ctr1_memory_bytes here

Sorry, I don't understand what you mean here. `ctr1_memory_bytes` is assigned here: https://github.com/cri-o/cri-o/pull/4064/commits/01813a89d989255209d28005c5e91cd8c3d72ccb#diff-103e5e134054b75239970c4fbec2f5e0R68

> I think these would be better if we just did

you're right, and it's better to take it one step further:

```bash
set -o pipefail
ctr1_mem=$(crictl stats -o json --id "$ctr1_id" | jq -e '.stats[0].memory.workingSetBytes.value')
ctr2_mem=$(crictl stats -o json --id "$ctr2_id" | jq -e '.stats[0].memory.workingSetBytes.value')
```

> maybe echoing them between

please see the updated commit.

Initially I wanted to change as little as possible. Ended up with more radical approach. Let me know what you think.